doc/Yang Model Guidelines.md

```diff
@@ -237,7 +231,7 @@ leaf family {
 
 
 
-### 9.  Use IETF data types for leaf type first if applicable (RFC 6021) . Declare new type (say Sonic types) only if IETF type is not applicable. All Sonic Types must be part of same header type or common yang model.
+### 8.  Use IETF data types for leaf type first if applicable (RFC 6021) . Declare new type (say SONiC types) only if IETF type is not applicable. All SONiC Types must be part of same header type or common YANG model.
```

Comment:
"Declare new type in common YANG model (say SONiC types) only if IETF type is not applicable."

We can remove "All SONiC types must be part of same header type or common YANG model."

Reply:
Agree.